random_menu.rb

```diff
@@ -0,0 +1,19 @@
+# Random Menu Generator for Boozy Desserts (Lunch) - Loraine Kanervisto
+# Thanks Pinterest for the inspiration
+
+#Creates an adjective array
+adjs = ["salty", "buttery", "spicy", "creamy", "frosty", "bitter", "sour", "sweet", "chunky", "hot", "cheesy", "burnt", "candied", "drizzled", "fluffy"]
```

Comment:
Some nice tasty words here. You can use literals for an array of all strings with %w[] (or %w() and %w{}, too), so like <code>%w[salty buttery spicy]</code> instead of <code>["salty", "buttery", "spicy"]</code> , but what you did works, too
(http://en.wikibooks.org/wiki/Ruby_Programming/Syntax/Literals#The_.25_Notation)